warehouse/recaptcha.py

```diff
@@ -0,0 +1,142 @@
+import collections
+import http
+from os import environ
+from urllib.parse import urlencode
+
+
+VERIFY_URL = "https://www.google.com/recaptcha/api/siteverify"
+
+
+# flake8: noqa
+class RecaptchaError(ValueError): pass
+class MissingInputSecretError(RecaptchaError): pass
+class InvalidInputSecretError(RecaptchaError): pass
+class MissingInputResponseError(RecaptchaError): pass
+class InvalidInputResponseError(RecaptchaError): pass
+class UnexpectedError(RecaptchaError): pass
+
+ERROR_CODE_MAP = {
+    "missing-input-secret": MissingInputSecretError,
+    "invalid-input-secret": InvalidInputSecretError,
+    "missing-input-response": MissingInputResponseError,
+    "invalid-input-response": InvalidInputResponseError,
+}
+
+ChallengeResponse = collections.namedtuple(
+    "ChallengeResponse", ("challenge_ts", "hostname")
+)
+
+
+class Service:
+    def __init__(self, request):
+        self.request = request
+
+    @property
+    def csp_policy(self):
+        if not self.enabled:
```

Comment:
I'm wondering if it wouldn't make sense to just remove this conditional and just always include this CSP on these pages. The only time this should skip the CSP is in development, and it doesn't seem harmful to have the CSP policy include this in development.